task/backend/scheduler/scheduler.go

```diff
@@ -0,0 +1,103 @@
+package scheduler
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb"
+	cron "gopkg.in/robfig/cron.v2"
+)
+
+// ID
```

Comment:
Might as well add your ideas as a TODO comment about how you don't want to import ID